src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -181,7 +181,7 @@ const MeetingForm = () => {
     await meetingManager.join({
       meetingInfo,
       attendeeInfo,
-      eventReporter: new NoOpEventReporter()
+      eventController: new DefaultEventController(configuration, logger, new NoOpEventReporter())
```

Comment:
Need to initialize the `configuration`, `logger` in this example, they are missing and confusing.

Reply:
Updated all cases